bokehjs/src/lib/models/widgets/datetime_range_slider.ts

```diff
@@ -0,0 +1,46 @@
+import tz from "timezone"
+
+import {AbstractSlider, AbstractRangeSliderView} from "./abstract_slider"
+import {TickFormatter} from "../formatters/tick_formatter"
+import * as p from "core/properties"
+import {isString} from "core/util/types"
+
+export class DatetimeRangeSliderView extends AbstractRangeSliderView {
+  override model: DatetimeRangeSlider
+}
+
+export namespace DatetimeRangeSlider {
+  export type Attrs = p.AttrsOf<Props>
+
+  export type Props = AbstractSlider.Props
+}
+
+export interface DatetimeRangeSlider extends DatetimeRangeSlider.Attrs {}
+
+export class DatetimeRangeSlider extends AbstractSlider {
+  override properties: DatetimeRangeSlider.Props
+  override __view_type__: DatetimeRangeSliderView
+
+  constructor(attrs?: Partial<DatetimeRangeSlider.Attrs>) {
+    super(attrs)
+  }
+
+  static {
+    this.prototype.default_view = DatetimeRangeSliderView
+
+    this.override<DatetimeRangeSlider.Props>({
+      format: "%d %b %Y %H:%M:%S",
+      step: 3600000,  // 1 hour.
```

Comment:
btw. One can use `_` for readability, e.g. `step: 3_600_000,`.

Reply:
Thanks.